### PR TITLE
LPS-141568 Unify color picker between Page Editor and StyleBooks

### PR DESCRIPTION
**Pull request by veroglez**

### CODE REVIEW

**Comment by veroglez** [comment]
ci:test:sf

**Comment by Tim-Cao** [comment]
ci:test:echo

**Comment by Tim-Cao** [comment]
Hi @veroglez and @sandrodw3, no regression issues found according to [ci:test:echo](https://testray.liferay.com/home/-/testray/case_results?testrayBuildId=1598153385&testrayRunId=1598153420)

File: modules/apps/style-book/style-book-web/src/main/resources/META-INF/resources/js/style-book-editor/FrontendTokenSet.js [hunk 1]
```diff
@@ -68,8 +68,12 @@ export default function FrontendTokenSet({frontendTokens, label}) {
 				...frontendTokensValues,
 				[name]: {
 					cssVariableMapping: cssVariableMapping.value,
-					name: tokenValues[value]?.name,
-					value: tokenValues[value]?.value || value,
+					value: config.tokenReuseEnabled
+						? tokenValues[value]?.value || value
+						: value,
```
Comment (p2kmgcl) [inline modules/apps/style-book/style-book-web/src/main/resources/META-INF/resources/js/style-book-editor/FrontendTokenSet.js:73]: I think we can simplify this with
`config.tokenReuseEnabled && tokenValues[value]?.value || value`
removing the `? :` statement.

Do you like it? Feel free to ignore me :3

**Review by p2kmgcl — COMMENTED** [review]
Just a tiny suggestion, everything else looks great :smile:

File: modules/apps/style-book/style-book-web/src/main/resources/META-INF/resources/js/style-book-editor/FrontendTokenSet.js (hunk 1 above)
Comment (veroglez) [reply modules/apps/style-book/style-book-web/src/main/resources/META-INF/resources/js/style-book-editor/FrontendTokenSet.js:73]: Sure!! I like it!

**Comment by veroglez** [comment]
ci:test:relevant

**Comment by ealonso** [comment]
ci:forward

**Pull request closed without merging**